### PR TITLE
demote WI_drawOnLnode() logging to debug

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

File: src/wi_stuff.c [hunk 1]
```diff
@@ -567,14 +567,14 @@ WI_drawOnLnode  // draw stuff at a location by episode/map#
     } 
   while (!fits && i!=2 && c[i]);
 
-  if (fits && i<2)
+  if (fits && c[i])
```
Comment (rfomin) [inline src/wi_stuff.c:570]: Why change this line? It's checked above, I don't think it's necessary. This code looks confusing, though.

Comment (fabiangreffrath) [reply src/wi_stuff.c:570]: If the function is called with `c == splat`, then the overflow may even occur if `i == 1`. It shouldn't pass the `fits` check though, but I find it more explicit this way.

Comment (rfomin) [reply src/wi_stuff.c:570]: We can remove `i < 2` and `c[i]` checks here, `fits` is enough. Perhaps we should rewrite this function, but Pr+, Choco and Eternity keep it as is. "Historical" code I guess 😄

Comment (fabiangreffrath) [reply src/wi_stuff.c:570]: Right, we can remove the `i!=2` check from the `while()` loop as well. I guess that the other ports kept this not for historic value, but for pragmatic "well, it works" approach. 😉

**Review by rfomin — APPROVED** [review]

**Pull request merged**